js/webserver.js

```diff
@@ -27,6 +27,18 @@ const bootstrapLogger = util.loggers.bootstrapLogger;
 const contentLogger = util.loggers.contentLogger;
 const childLogger = util.loggers.childLogger;
 
+function sanitizeIps(ipAddresses) {
+  let set = new Set();
+  ipAddresses.forEach((ipAddress)=> {
+    if (typeof ipAddress == 'string') {
+      set.add(ipAddress);
```

Comment:
I'm wondering what we should do if we get synonymous addresses here, e.g. `[ 'localhost', '127.0.0.1' ]` or `[ 'rs22', 'rs22.rocketsoftware.com' ]`? The fact that it's a Set won't help us, and we'll try to bind to the same address twice.

Reply:
Did you test what happens if you do that? Think what you want the outcome to be, and figure out how to achieve the outcome. I suspect you might get an error if you try to bind separately to synonyms (such as localhost and ::1 or 127.0.0.1).

If you do get errors, I suggest issuing an informative error message. As to whether you proceed or abort after that, that's a discussion.

Reply:
The use of Set drew my attention. It looks like we're trying to provide a feature but didn't finish the job.  If we really care to remove synonymous addresses for the user, then a Set of string hostnames won't be enough, we'd also need to resolve them and see that they're actually unique.  Or we might not care and just let the user make sure there're no synonyms. Then again we don't need a Set here.